reactjs/src/index.js

```diff
@@ -19,16 +19,21 @@ const CUSTOM_OPERATORS = [
 function Index() {
   const [operators, setOperators] = useState([]);
   const [displayHistory, setDisplayHistory] = useState(false);
+  const [displayDeleteButtonOnHistory, setDisplayDeleteButtonOnHistory] = useState(false);
```

Comment:
Nit: apply `./node_modules/.bin/prettier --write "**/*.js"` so that CI passes.
```suggestion
  const [
    displayDeleteButtonOnHistory,
    setDisplayDeleteButtonOnHistory
  ] = useState(false);
```